src/singlePoem/SinglePoem.js

```diff
@@ -23,7 +25,9 @@ function SinglePoem({ poems }) {
     />
   ) : null
 
-  return <div>{poemCard}</div>;
+  if (selectedPoem) {
```

Comment:
Id do something like selectedPoem && <div>{poemCard}</div> - this kind of conditionally rendering is much more common in react!